apisix/discovery/nacos/init.lua

```diff
@@ -225,6 +224,12 @@ local function iter_and_add_service(services, values)
         else
             up = conf
         end
+        local nacos_name_from_args = (up.discovery_args and up.discovery_args.name)
+             or default_nacos_name
```

Comment:
I remember other service discovery implementations using different rules on service name to handle multiple registries instead of using discovery_args, please follow that similar.

https://apisix.apache.org/docs/apisix/discovery/kubernetes/#multi-cluster-mode-query-interface